### PR TITLE
chore: bump to Angular 8.0.0

### PR DESCRIPTION
**Pull request by hsuanxyz**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[x] Other... Please describe:
```

## What is the current behavior?

Issue Number: #3514

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by lppedd** [comment]
I would suggest waiting for 8.1 before merging

**Comment by vthinkxie** [comment]
> I would suggest waiting for 8.1 before merging. The first beta already fixed _a lot_ of bugs.

yes, we will try to release ng-zorro-antd 8.0 after angular 8.1 released

File: components/button/nz-button.component.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class NzButtonComponent implements AfterContentInit, OnInit, OnDestroy, O
   readonly el: HTMLElement = this.elementRef.nativeElement;
   private iconElement: HTMLElement;
   private iconOnly = false;
-  @ViewChild('contentElement') contentElement: ElementRef;
+  @ViewChild('contentElement', { static: true }) contentElement: ElementRef;
```
Comment (vthinkxie) [inline components/button/nz-button.component.ts]: is static true required here

**Comment by lppedd** [comment]
See this document for Ivy breaking changes, so that you might have a better overview of the things to change in the future : )
https://docs.google.com/document/d/1Dije0AsJ0PxL3NaeNPxpYDeapj30b_QC0xfeIvIIzgg/preview

**Pull request merged**